bin/xud

```diff
@@ -134,6 +134,11 @@ const { argv } = require('yargs')
       type: 'number',
       alias: 'p',
     },
+    'raiden.cltvDelta': {
```

Comment:
cltv is not really a term used by raiden.
Maybe `lockDuration` ?

Reply:
I was thinking of making this a global config parameter (rather than each lnd client and raiden having its own) that's specified in minutes or hours which is then calculated into blocks at runtime. I'm fine with `lockDuration`, it would just be the delay that we require for the "hop" between chains.